src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -658,7 +665,8 @@ internal static bool IsValidAuthenticationTypeValue(SqlAuthenticationMethod valu
                 || value == SqlAuthenticationMethod.ActiveDirectoryManagedIdentity
                 || value == SqlAuthenticationMethod.ActiveDirectoryMSI
                 || value == SqlAuthenticationMethod.ActiveDirectoryDefault
-                || value == SqlAuthenticationMethod.NotSpecified;
+                || value == SqlAuthenticationMethod.NotSpecified
```

Comment:
missing ';'

Reply:
Its not since line 669 extends this comparison with
|| value == SqlAuthenticationMethod.SqlCertificate;